scripts/start.js

```diff
@@ -22,8 +22,18 @@ if (process.env.npm_config_schemaPath) {
   } catch (e) {
     console.error("Unable to load GQL schema from provided path", e);
   }
-  process.env.REACT_APP_SCHEMA_STRING = schemaString || "";
+  if (schemaString) {
+    process.env.REACT_APP_SCHEMA_STRING = schemaString;
+  }
+}
+if (process.env.npm_config_enableGQLMockServer) {
```

Comment:
this allows the user to set `REACT_APP_SCHEMA_STRING` by setting the `-schemaPath` parameter in `npm run start` to a file path containing your custom schema string. I just pushed some commits that will set this variable using the cmdrc library instead since we plan on rolling an unejected version of CRA and changes in this file will be reset